src/cdk/table/table.ts

```diff
@@ -218,7 +218,7 @@ export class CdkTable<T> implements CollectionViewer {
    * Check if the header or rows have changed what columns they want to display. If there is a diff,
    * then re-render that section.
    */
-  private _renderUpdatedColumns() {
+  private _renderUpdatedColumns(): void {
```

Comment:
Private methods in general don't need to have an explicit type.

Reply:
I'm trying to create consistency on typing.

Reply:
The convention at the moment is that if it goes into the docs, it should be typed (so if it's public, doesn't start with an underscore and it's not an Angular lifecycle hook). IMO having to add the types everywhere might cause some churn.